rkt/run.go

```diff
@@ -81,6 +84,7 @@ func init() {
 	cmdRun.Flags().BoolVar(&flagNoOverlay, "no-overlay", false, "disable overlay filesystem")
 	cmdRun.Flags().BoolVar(&flagPrivateUsers, "private-users", false, "run within user namespaces.")
 	cmdRun.Flags().Var(&flagExplicitEnv, "set-env", "environment variable to set for apps in the form name=value")
+	cmdRun.Flags().Var(&flagEnvFromFile, "set-env-file", "path to an environment variables file.")
```

Comment:
To stay consistent, the help string should not finish with a dot.

I see that --interactive and --private-users have a dot. To fix in another PR?